spec.html

```diff
@@ -30489,7 +30493,8 @@ <h1>String.prototype.substring ( _start_, _end_ )</h1>
           1. Let _finalEnd_ be min(max(_intEnd_, 0), _len_).
           1. Let _from_ be min(_finalStart_, _finalEnd_).
           1. Let _to_ be max(_finalStart_, _finalEnd_).
-          1. Return the String value whose length is _to_ - _from_, containing code units from _S_, namely the code units with indices _from_ through _to_ - 1, in ascending order.
+          1. If _from_ = _to_, return the empty String.
```

Comment:
Why include this step?

Reply:
I think it made more sense in an earlier iteration; removing.